test-packages/framework/src/framework.ts

```diff
@@ -97,9 +102,89 @@ export class TestFramework implements TestFrameworkAPI {
     }
     const offset = this.getOffset(content);
     this.offset = offset;
-    content = content.replace(/⇶/, "");
+    content = content.replace(new RegExp(CURSOR_ANCHOR, "g"), "");
     await writeFile(filePath, content);
   }
+
+  /**
+   * Updates file content
+   * @param filePath - path of file to update
+   * @param newText - new text to be written to the file
+   * @param options - describes how the file should be updated
+   *    - insertAfter: if provided the text is inserted after that fragment
+   *    - replaceText: if provided the new text replaces that fragment
```

Comment:
please also add doc for

`insertBefore`
and `doUpdatesAfter`

Reply:
done